eks-distro-base/build_image.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Do you want to rename this file something more accurate? `update_base_image.sh`?